### ISSUE TITLE
Add thickened tricuspid valve cusp

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10553]
**Preferred term label:**
Thickened tricuspid valve cusp

**Synonyms**
-

**Definition (free text, please give PubMed ID)**
An abnormally increased thickness of a leaflet of the tricuspid valve. PMID28706876

**Parent term (use hpo.jax.org/app)**
Abnormal tricuspid valve morphology HP:0001702

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Tricuspid stenosis ICD-10:I36.2

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw @azankl
Adding new term: 
- ID: HP:6001068
- label: Thickened tricuspid valve cusp
- parents: [Abnormal tricuspid valve morphology (HP:0001702)](https://hpo.jax.org/app/browse/term/HP:0001702)
- pmid: PMID:28706876
- def: An abnormally increased thickness of a leaflet of the tricuspid valve.
- comment: 
- orcid: orcid:0009-0007-1020-5412
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10553
- synonyms:
